.drone.yml

```diff
@@ -1,7 +1,21 @@
 build:
-  image: ensime/ensime:v1.x-cache
-  pull: true
-  commands:
-    - pip install virtualenv python-coveralls
-    - make test
+  quality:
+    image: ensime/ensime:v1.x-cache
+    pull: true
+    commands:
+      - pip install --upgrade virtualenv
+      - make lint
+      - make format ;
+        if $(! git diff --exit-code --quiet) ; then
+          echo "Code formatting does not meet the project's standards:" ;
```

Comment:
The trouble with Git hooks is we have to ask everyone to install them locally. Seems unlikely to happen in a public project and the extra setup work might be *more* of a barrier to contribution than a CI failure.

I used this initially as what seemed to me "simplest thing that would work"—we could look at other things that might be a bit nicer experience for contributors, like [Scrutinizer][1] or [Landscape][2], but that's more stuff to set up and they use different tools & configuration than what is already in place so there would be a few changes (I'm not opposed to considering that route though).

I took this `echo` straight from ensime-server's main CI run by the way, although in that case Scalariform runs automatically on `sbt compile` so there's a good chance you would have noticed it before preparing a pull request. Agree that our expectations (and available `make` targets, etc.) should definitely be documented in contributing docs, however we end up implementing the checks.

[1]: https://scrutinizer-ci.com/docs/guides/python/automated-code-reviews
[2]: https://landscape.io/